homeassistant/components/airzone/select.py

```diff
@@ -32,27 +32,16 @@ class AirzoneSelectDescriptionMixin:
     """Define an entity description mixin for select entities."""
 
     api_param: str
-    options_dict: dict[str, int]
 
 
 @dataclass
 class AirzoneSelectDescription(SelectEntityDescription, AirzoneSelectDescriptionMixin):
     """Class to describe an Airzone select entity."""
 
 
-GRILLE_ANGLE_DICT: Final[dict[str, int]] = {
-    "90º": GrilleAngle.DEG_90,
-    "50º": GrilleAngle.DEG_50,
-    "45º": GrilleAngle.DEG_45,
-    "40º": GrilleAngle.DEG_40,
-}
+GRILLE_ANGLE_OPTIONS: Final[list[str]] = [str(opt.value) for opt in GrilleAngle]
```

Comment:
I think we should keep the dicts so that the integration is in control of the options as they need to match the translation strings.

Reply:
Done in b818a2674d6bdec7792fbb76e00b3a3a261d663b